tools/validators/instance_validator/validate/instance_parser.py

```diff
@@ -42,34 +54,20 @@
                            syaml.Map({
                                'type': syaml.Str(),
                                'id': syaml.Str(),
-                               syaml.Optional('connections'):
-                                 syaml.MapPattern(syaml.Str(),
-                                                  syaml.Str()) |
-                                 syaml.Seq(
-                                     syaml.MapPattern(syaml.Str(),
-                                                      syaml.Str())),
+                               # TODO(b/166472270): revisit connections
```

Comment:
Always explain the "why". ex: "revisit connection syntax validation.  Current code might not follow the spec."

Reply:
done